crd/apis/gcpfirewall/v1alpha1/types.go

```diff
@@ -93,14 +92,24 @@ type GCPFirewallSpec struct {
 // sub-fields are combined using a logical OR operation. If all the sub-fields
```

Comment:
This is no longer fully accurate. We might want to update this -- we can call it a cross-product of source and destination.

Reply:
Updated. See if this is better.